cdc/log.cc

```diff
@@ -805,26 +786,36 @@ class transformer final {
 
                         ++pos;
                     }
-
-                    auto ttl = process_cells(r.row().cells(), column_kind::regular_column, log_ck, pikey, pirow);
-                    const auto& marker = r.row().marker();
-                    if (marker.is_live() && marker.is_expiring()) {
-                        ttl = marker.ttl();
-                    }
-
+                    
                     operation cdc_op;
                     if (r.row().deleted_at()) {
                         cdc_op = operation::row_delete;
-                    } else if (marker.is_live()) {
-                        cdc_op = operation::insert;
+                        if (pirow) {
```

Comment:
note: this is out of scope of this patch, but it would be nice to add a comment somewhere, which explains what `pirow` means (if anyone's curious, it stands for pre-image row)

Reply:
Right. Moreover, since you have other things called preimage_something, it would make sense just to call this variable preimage_row, and no comment would have been necessary.